src/components/Navbar/Navbar.js

```diff
@@ -1,22 +1,23 @@
 import React, { Component } from 'react';
 
+import s from './Navbar.module.css';
```

Comment:
Consider renaming "s" to "styles" to be more descriptive.

Reply:
/fxd